cli/pcluster/update/update_command.py

```diff
@@ -206,3 +207,17 @@ def _restore_desired_size(cfn_params, stack_name, scheduler):
     # If there are compute nodes running, preserve current desired capacity (if possible)
     if int(cfn_params["MinSize"]) <= desired_capacity <= int(cfn_params["MaxSize"]):
         cfn_params["DesiredSize"] = str(desired_capacity)
+
+
+def _check_cluster_models(base_config, target_config):
+    if target_config.cluster_model == ClusterModel.SIT:
+        if base_config.cluster_model == ClusterModel.HIT:
+            LOGGER.error(
+                "The configuration file '{config_file}' must be converted before proceeding with the update.\n"
+                "Please run the following command:\n"
+                "pcluster-utils convert_config_file {config_file} -o {new_file}\n"
+                "then retry with:\n"
+                "pcluster update -c {new_file} {cluster_name}\n".format(
```

Comment:
Here's my initial suggestion:

> The configuration file '{config_file}' must be converted to the latest format with support for multiple queues before proceeding with the update.
> Please run the following command:
> ...
> Then retry with your converted configuration file by running the following command:
> ...

What do you think?

Reply:
Looks perfect to me! Thanks!